app/models/comment.rb

```diff
@@ -12,9 +12,9 @@ class Comment < ApplicationRecord
   COMMENTABLE_TYPES = %w[Article PodcastEpisode].freeze
   TITLE_DELETED = "[deleted]".freeze
   TITLE_HIDDEN = "[hidden by post author]".freeze
-  MAX_USER_MENTIONS = 7 # Explicitly set to 7 to accomodate DEV Top Seven Posts
-  # The date that we began limiting the nubmer of user mentions in a comment.
-  MAX_USER_MENTION_LIVE_AT = Time.utc(2021, 3, 11).freeze
+  MAX_USER_MENTIONS = 7 # Explicitly set to 7 to accommodate DEV Top 7 Posts
+  # The date that we began limiting the number of user mentions in a comment.
+  MAX_USER_MENTION_LIVE_AT = Time.utc(2021, 3, 12).freeze
```

Comment:
I chose to not make this a SiteConfig configurable date because I don't expect we'll change it again -- and we definitely don't want this to be configurable for Forem admins, so that also factored into my decision.

The other thing to consider is that this would be a non-UI SiteConfig, which feels a bit hacky/like an antipattern, so that was another reason why I chose to just bump it here 👍🏽